src/python/zquantum/qcbm/target_thermal_states_test.py

```diff
@@ -0,0 +1,246 @@
+import unittest
```

Comment:
This file should be moved to `tests/zquantum/qcbm/`

Reply:
Also, we have moved to using `pytest` instead of `unittest`. If you'd like help rewriting the following tests, I can help :)